agentai/conversation.py

```diff
@@ -13,8 +13,11 @@ class Message(BaseModel):
 
 
 class Conversation:
-    def __init__(self, history: List[Message] = [], id: Optional[str] = None, max_history_tokens: int = 200):
+    def __init__(
+        self, history: List[Message] = [], id: Optional[str] = None, max_history_tokens: int = 200, model: str = "gpt2"
```

Comment:
Let's default to `gpt3.5-turbo` and use the corresponding encoder, tokenizer as well everywhere. This "gpt2" is confusing.